cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -290,9 +297,122 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 				continue
 			}
 			// TODO: report back the information below, which is necessary to find and use the image
-			// imageName := t.ImageName
+			// imageName := t.BlobName
 			// projectID := gcp.GetProjectID()
 
+		case *target.AzureImageTargetOptions:
+			ctx := context.Background()
+
+			if impl.AzureCreds == nil {
+				r = append(r, errors.New("osbuild job has org.osbuild.azure.image target but this worker doesn't have azure credentials"))
+				continue
+			}
+
+			c, err := azure.NewClient(*impl.AzureCreds, options.TenantID)
+			if err != nil {
+				r = append(r, err)
+				continue
+			}
+			log.Print("[Azure] 🔑 Logged in Azure")
```

Comment:
Not an issue, just a comment probably for further discussion. This kind of logging is done in GCP's and AWS's case in the `internal/upload/...` implementation, not in worker's job implementation. In case of GCP, I followed what was already implemented for AWS, but I can see that Azure never had any logging in the internal upload implementation. We should probably decide what we want and should make it consistent for all types of upload targets.

Reply:
Yeah, my thinking is that `internal/upload/...` should serve mainly as a library and thus not log anything... But yeah, we should have a discussion. :)